### PR TITLE
Fix developers API

### PR DESCRIPTION
**Pull request by Suvitruf**
https://github.com/facundoolano/google-play-scraper/issues/284#issuecomment-520255165

### CODE REVIEW

**Comment by Suvitruf** [comment]
@facundoolano, even if it's working, probably it doesn't feet the rest project code style, etc. Would be nice to see any recommendations/suggestions (:

**Comment by facundoolano** [comment]
So is it working? Can you summarize what the implementation is now? I see that you had to increase the mocha timeout, did the method get much slower now?

**Comment by Suvitruf** [comment]
> 
> 
> So is it working? 

Yeh, I use this implementation right now for our project, and it works. 

> Can you summarize what the implementation is now?

1. We load developer page. It could be in different format:

   - With string id, e.g. https://play.google.com/store/apps/developer?id=Google%20LLC
   -  Int id, e.g. https://play.google.com/store/apps/dev?id=5700313618786177705

2. Get [cluster link](https://github.com/facundoolano/google-play-scraper/blob/dev/lib/search.js#L61) from button, e.g. `https://play.google.com/store/apps/collection/cluster?clp`.
3. Send requests like in [`search` ](https://github.com/facundoolano/google-play-scraper/blob/dev/lib/search.js).
And receive response, but it is not html, and not fully valid json (there are some strange symbols in the beginning of the response). We [cut](https://github.com/facundoolano/google-play-scraper/blob/dev/lib/search.js#L47) them, and then parse it as json and [parse apps using `REQUEST_MAPPINGS`](https://github.com/facundoolano/google-play-scraper/blob/dev/lib/search.js#L49).

> I see that you had to increase the mocha timeout, did the method get much slower now?

I've increased it, 'cause i passed `fullDetail: true`

**Comment by Suvitruf** [comment]
> did the method get much slower now?

I haven't tested it. I mean, for me, there were no choice, 'cause there is no working alternative. I can check it later to see, if I can optimize some things.

**Comment by xenio** [comment]
It is working for me. I get all my apps as developerID

**Comment by facundoolano** [comment]
I'm inclined to merge this now, but there's one of the developer tests that is still failing, the one that checks that multiple pages of results are fetch, can you fix that one too?
By the looks of the website, there are less than 100 apps from Google LLC, so I guess it makes sense that it fails if we ask for >= than 100

**Comment by Suvitruf** [comment]
Sure, I'll going to check it on my day off.

**Comment by facundoolano** [comment]
Looks like two method fail now, I wonder if the recent changes in the website have broken this as well?

**Comment by icarcal** [comment]
Hey, guys.
Google seems to have two different urls:
- The first one:
  - https://play.google.com/store/apps/dev?id=7908612043055486674&hl=en (probably for some opt-in setting)
- The second one
  - https://play.google.com/store/apps/developer?id=WhatsApp+Inc.&hl=en

I think that we need to add a fallback to this for now, for example, search for the new one with the devId, if the URL is not found, check for the second URL.

The only problem with this approach is that both methods can fail, leaving you with no information about the developer apps.

Unfortunatelly this seems like another breaking change :disappointed:

I'll try to come up with something for this case too.

**Comment by icarcal** [comment]
@facundoolano @Suvitruf check the PR #344 
Finally the building is passing :tada:

**Comment by facundoolano** [comment]
fixed by #344
